packages/wonder-blocks-modal/components/modal-header.test.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from "react";
+import {shallow} from "enzyme";
+
+import expectRenderError from "../../../utils/testing/expect-render-error.js";
+import ModalHeader from "./modal-header.js";
+
+describe("ModalHeader", () => {
+    test("using only `subtitle` should render the header", () => {
+        const wrapper = shallow(
+            <ModalHeader title="Title" subtitle="Subtitle" />,
+        );
+
+        // we make sure the component is rendered
+        expect(wrapper.exists()).toBe(true);
```

Comment:
Nit: we try to use AAA (Arrange, Act, Assert) style testing.  Search for `// Arrange` to see examples.  In this case the "Arrange" and "Act" are one in the same so you could `// Arrange, Act` and then `// Assert` before the `expect`.